_pages/en_US/saving-blobs/Automatic blobsaving.md

```diff
@@ -0,0 +1,12 @@
+Please note that this works only for Procursus jailbreak users, meaning unc0ver and base checkra1n (not odysseyra1n) are not supported {: .notice--danger}
+
+## Saving blobs automatically
+1. Search for and download "shshd"
+2. Done! now blobs will automatically be saved when you download shshd and every sunday. The blobs are available at the [TSS saver website](https://tsssaver.1conan.com/v2/), in order to retrieve them, go to "retrieve" and put in your ECID
```

Comment:
```suggestion
2. Done! now blobs will automatically be saved when you download shshd, every time you rejailbreak, and every week after that. The blobs are available at the [TSS saver website](https://tsssaver.1conan.com/v2/), in order to retrieve them, go to "retrieve" and put in your ECID
```
v1.0.1 now saves them every week starting from the time your device booted. rather than specifically on Sundays.

Reply:
true